src/components/PriceDisplay/PriceDisplay.js

```diff
@@ -59,8 +62,14 @@ class PriceDisplay extends React.PureComponent<Props> {
 			symbol,
 			preSymbol,
 			paymentRequestUrl,
+			invoiceTimeLeftSeconds,
 		} = this.props;
 
+		let isTimerShown = false;
```

Comment:
Mentioned in the above comment, this is part of the file/flow I think should be rethought.
Price Display should continue to show the `price` information, even for BIP70 payments.  Then another component should be made to show if it's a payment request and the timer.